### PR TITLE
Add support for the ipsets section

### PR DESCRIPTION
**Pull request by jof**
Addresses #21

### CODE REVIEW

File: main.go [hunk 1]
```diff
@@ -30,6 +30,7 @@ var (
 		"extraDNSRecords": handleArray(),
 		"grants":          handleArray(),
 		"groups":          handleObject(),
+		"ipsets":          handleArray(),
```
Comment (clstokes) [inline main.go:33]: `ipsets` [is an object](https://tailscale.com/kb/1387/ipsets#syntax), not an array, so it should use `handleObject()` instead. Can you make that change and also update [testdata](https://github.com/tailscale-dev/tailscale-acl-combiner/tree/main/testdata) to ensure tests pass?

**Review by clstokes — CHANGES_REQUESTED** [review]
Added a comment about the proposed change and requested updates to ensure tests pass.

File: main.go (hunk 1 above)
Comment (jof) [reply main.go:33]: Yup -- I guess that is my derpy mistake; I was just skimming `handleArray()` and `handleObject()` and I thought it was about recombination of the values for cases where the keys match.

Will update.

**Review by clstokes — APPROVED** [review]
Looks great, thanks!

**Pull request merged**